app/views/directives/create-secret.html

```diff
@@ -0,0 +1,310 @@
+      <div class="help-block">
+        Docker configuration file with credentials and configuration to the secured registry.
+      </div>
+    </div>
+  </div>
+
+  <div ng-if="'serviceaccounts' | canI : 'update'">
+    <div class="form-group">
+      <div class="checkbox">
+        <label>
+          <input type="checkbox" ng-model="linkSecret">
+          Use this secret automatically for other builds.
+        </label>
+      </div>
+    </div>
+
+    <div ng-if="!serviceAccountToLink && linkSecret">
+      <div class="form-group">
+        <label for="serviceAccount" class="required">Service Account</label>
+        <ui-select required ng-model="newSecret.pickedServiceAccountToLink">
+          <ui-select-match placeholder="Service Account Name">{{$select.selected}}</ui-select-match>
+          <ui-select-choices repeat="sa in (serviceAccountsNames | filter : $select.search)">
+            <div ng-bind-html="sa | highlight : $select.search"></div>
+          </ui-select-choices>
+        </ui-select>
+      </div>
+
+      <div class="form-group" >
+        <label class="required">Link as</label>
+
+        <div class="form-group">
+          <div class="checkbox">
+            <label>
+              <input type="checkbox" ng-model="newSecret.linkAs.secrets" ng-checked="newSecret.linkAs.secrets">
+              Link with {{newSecret.pickedServiceAccountToLink}} service account as a <b>source</b> secret.
+              <span class="help action-inline">
+                <a href="">
+                  <i class="pficon pficon-help" aria-hidden="true"
+                  data-toggle="tooltip" data-original-title="Pods using this service account will mount the content of secret into their containers for pulling sources during build time.">
+                </i>
+                </a>
+              </span>
+            </label>
+          </div>
+        </div>
+
+        <div class="form-group">
+          <div class="checkbox">
+            <label>
+              <input type="checkbox" ng-model="newSecret.linkAs.imagePullSecrets" ng-checked="newSecret.linkAs.imagePullSecrets">
+              Link with {{newSecret.pickedServiceAccountToLink}} service account as a <b>image pull</b> secret.
+              <span class="help action-inline">
+                <a href="">
+                  <i class="pficon pficon-help" aria-hidden="true"
+                  data-toggle="tooltip" data-original-title="Pods using this service account will use provided creadentials to pull images for the pod’s containers.">
+                </i>
+                </a>
+              </span>
+            </label>
+          </div>
+        </div>
+        <div class="help-block">
+          Linking a secret enables a service account to automatically use that secret for some forms of authentication.
+          <a href="{{'managing_secrets' | helpLink}}" target="_blank"><span class="learn-more-inline">Learn more<i class="fa fa-external-link" aria-hidden="true"></i></span></a>
```

Comment:
We should add a non-breaking space between "Learn more" and the icon (here and other learn more links).

```html
Learn more&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i>
```